modules/@apostrophecms/piece-type/index.js

```diff
@@ -96,6 +96,13 @@ module.exports = {
       }
     }
   },
+  moreMenu: {
```

Comment:
Adding this as a cascade makes sense. Let's rename it to be less tied to the UI and more about the purpose. Something like `managerActions` (still a little about the UI, but less specific), simply `operations`, `utilityOperations` (maybe my favorite).

Reply:
Like `utilityOperations` too because it's consistent with batchOperations 👍🏼